### PR TITLE
instantiate docs and initial attempt at glossary

### PR DESCRIPTION
**Pull request by brownsarahm**
We need to get researcher input on how much variation and structure they want to repeat to finalize, but this is a starting point. Look mostly at `docs/source/glossary.rst` and `docs/source/technical-overview.rst`

### CODE REVIEW

File: docs/source/glossary.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    a deployment of the statistical perceptions platform consisting contained in
+    one Heroku app. An instance of the app consists of one set of databases.
+
+  experiment
+    a template for studies. a given experiment can be run many times with
+    different settings for different studies. an experiment consists of a set
+    of items of a given type but may have different values of parameters
+
+  app
+    FIXME as in app1 and app2
```
Comment (QuantumScrewdriver) [inline docs/source/glossary.rst:22]: a website, on which experiments can be either designed and saved, or on which experiments can be run by survey respondents

Comment (brownsarahm) [inline docs/source/glossary.rst:22]:
```suggestion
    a website, on which experiments can be either designed and saved, or on which experiments can be run by survey respondents
```

File: docs/source/glossary.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Glossary
+--------
+
+Key terms used throughout the project
+
+.. glossary::
+  :sorted:
+
+  study
+    a fully configured experiment for a participant that consists of FIXME
```
Comment (brownsarahm) [inline docs/source/glossary.rst:10]:
```suggestion
    a fully configured experiment for a participant that consists of qualtrics surveys and an experiment on this platform including a specific cohort of subjects
```

File: docs/source/glossary.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  instance
+    a deployment of the statistical perceptions platform consisting contained in
+    one Heroku app. An instance of the app consists of one set of databases.
+
+  experiment
+    a template for studies. a given experiment can be run many times with
+    different settings for different studies. an experiment consists of a set
+    of items of a given type but may have different values of parameters
```
Comment (brownsarahm) [inline docs/source/glossary.rst:19]:
```suggestion
    a group of items with all parameters set. 
```

File: docs/source/technical-overview.rst [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+another metric, next button
+parameters: ROC?/ tbd, labeling
+records: preferred location of tradeoff
+
+
+**Static Information**
+for added information between prompts
+control: next button
+paramters: text and images or link to video to embed
+records: nothing
```
Comment (brownsarahm) [inline docs/source/technical-overview.rst:54]:
```suggestion
**Information**
for added information between prompts, noninteractive, may depend on previous responses
control: next button
paramters: text and images or link to video to embed
records: nothing
```

**Pull request merged**